Sources/SwiftSyntaxMacroExpansion/MacroSystem.swift

```diff
@@ -951,17 +951,17 @@ private extension AccessorBlockSyntax {
 }
 
 private extension String {
-  /// Add any non-comment trivia from `node` before/after this string.
+  /// Add any trivia from `node` before/after this string.
   ///
-  /// We need to do this because the macro is responsible for copying trivia
-  /// from the freestanding macro to the generated declaration.
-  ///
-  /// Essentially, what we want to keep any empty newlines in front of the
-  /// freestanding macro that separate it from the previous declarations.
+  /// We need to do this because we are replacing the entire macro expansion
+  /// declaration / expression with the expanded source but the “Inline Macro”
+  /// refactoring only replaces the node’s content, and doesn’t touch its trivia.
   func wrappingInNonCommentTrivia(from node: some SyntaxProtocol) -> String {
```

Comment:
Still called `wrappingInNonCommentTrivia`

Reply:
Thanks for catching these things, as always